### ISSUE TITLE
Whici allele is Risk allele for EUR?

### ISSUE THREAD

**Issue by Haoted** [gwas-partitioning/bnmf-clustering#2]
Hi, @hyunkyungclairekim, thank you for your wonderful work! 

I am just a new beginner. I noticed that in the supplementary table 5 of [Smith, Deutsch et al Nature Medicine 2024](https://www.nature.com/articles/s41591-024-02865-3), you didn't show the risk allele for EUR. How can we get that information? I also found that in your folder bnmf-clustering/Smith_Deutsch_NatureMedicine_2024/EUR/, although there are risk alleles, the number of variants didn't match that in the supplementary Table 5 from the paper.

I am very confused about it, it's very kind of you to give me some instructions. Thanks in advance!

**Comment by kirk-broad** [comment]
Hi @Haoted, I apologize for the way overdue response; there were issues with the repo alerts not reaching my email. I plan to keep this page better monitored going further.

The cluster weight files uploaded to _Smith_Deutsch_NatureMedicine_2024_ only include the SNPs above the weight cutoff, which we calculate specifically for each iteration of clustering. As described in the paper, we find a threshold that optimizes the signal-to-noise ratio.

To summarize, Supp Table 5 includes all of the SNPs that went into the European clusters, but only those above the weight threshold are used to calculate the cluster pPS.
